samples/bluetooth/peripheral_uart/prj.conf

```diff
@@ -20,8 +20,8 @@ CONFIG_BT=y
 CONFIG_BT_PERIPHERAL=y
 CONFIG_BT_DEVICE_NAME="Nordic_UART_Service"
 CONFIG_BT_DEVICE_APPEARANCE=833
-CONFIG_BT_MAX_CONN=2
-CONFIG_BT_MAX_PAIRED=2
+CONFIG_BT_MAX_CONN=1
```

Comment:
Why it is reduced to one connection ?

Reply:
Because of the way the sample is written it doesn't really support multiple connections.
I added explanation in the commit message. Is there a specific reason why it needs to be 2 connections?

Reply:
Probably there is not any reason.